app/views/feedback/new.html.haml

```diff
@@ -0,0 +1,6 @@
+%section.form_wrapper
+  = simple_form_for(@form, url: feedback_path) do |f|
+    = f.hidden_field :url
+    = f.input :email, as: :email
+    = f.input :comment, as: :text
+    = f.button :submit, 'Send Feedback', role: 'button'
```

Comment:
Thanks for fixing the class and id issues but this page still needs an h1. As does the create page.